entity-management-web/src/main/java/eu/europeana/entitymanagement/batch/DummyJobConfigurer.java

```diff
@@ -0,0 +1,62 @@
+package eu.europeana.entitymanagement.batch;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.JobParametersIncrementer;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.repeat.RepeatStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+/**
+ * Stub job configurer to test that Spring Batch integration works
+ */
+@Component
+public class DummyJobConfigurer {
```

Comment:
I suppose that this class is not needed or?

Reply:
Indeed, this is a _temporary_ class which was introduced to test the Spring Batch integration. 

It definitely won't be in the final product.